src/clients/add-on.rb

```diff
@@ -67,7 +68,7 @@ def run_GUI
       end
 
       # initialize target to import all trusted keys (#165849)
-      Pkg.TargetInitialize("/")
+      Pkg.TargetInitialize(Installation.destdir)
```

Comment:
BTW is better to use `Installation.destdir` or `WFM.scr_root` ? The later is more generic and also does not need dependency on Installation module.

Reply:
- Technically there is no much difference between these two, but `Installation.destdir` is already used at many places, we should be consistent
- The `Installation` module is part of the base`yast2` package, so no extra dependency.
- There is a tiny difference, the `Installation.destdir` is during installation set to `/mnt` from the very beginning ([link](https://github.com/yast/yast-yast2/blob/master/library/control/src/modules/Installation.rb#L212-L214)), but the SCR is chrooted at the end of installation. So I think the meaning of `Installation.destdir` logically better fits into this scenario.